docs/Documentation/Applications/comsol.md

```diff
@@ -15,18 +15,26 @@ Extensive documentation is available in the menu: **Help > Documentation**. For
 For instructional videos, see the [COMSOL website](https://www.comsol.com) Video Gallery.
 
 ## Building Models in the COMSOL GUI
-Before beginning, it is good practice to check the license status with:
```

Comment:
I can't put a comment on a line you didn't edit, but I think this should be removed since it doesn't work on Kestrel: 
"Issue the command `lmstat.comsol` to see current license status and COMSOL modules available." in the introductory paragraph. I think the next section explaining how to check license availability on Kestrel is sufficient.